pypopquiz/io.py

```diff
@@ -31,6 +44,54 @@ def total_duration(clips: List[Dict]) -> int:
     return sum((get_interval_duration(item["interval"]) for item in clips))
 
 
+def substitute_variables(data: Dict) -> None:
```

Comment:
Maybe we can put all the variables related functions in a separate file to make the I/O file a bit less crowded?